pkg/cloud/scope/cluster.go

```diff
@@ -146,6 +146,14 @@ func (s *ClusterScope) Region() string {
 	return s.AWSCluster.Spec.Region
 }
 
+// ControlPlaneLoadBalancerScheme returns the Classic ELB scheme (public or internal facing)
+func (s *ClusterScope) ControlPlaneLoadBalancerScheme() infrav1.ClassicELBScheme {
+	if s.AWSCluster.Spec.NetworkSpec.ControlPlaneLoadBalancerScheme == "" {
+		return infrav1.ClassicELBSchemeInternetFacing
+	}
+	return s.AWSCluster.Spec.NetworkSpec.ControlPlaneLoadBalancerScheme
```

Comment:
There are downsides to applying defaults here: for example, the default values are not discoverable. With v1alpha2,  `AWSCluster` is a CRD, and at some point we can use [CRD defaulting](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#defaulting). However, applying the default here has precedent (https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/1126).

Reply:
There should be a tracking issue upstream to raise the minimum Kubernetes version to 1.16, which brings support for defaulting. For now, making it explicit in a comment should be ok

Reply:
Not sure what the call-out is here.. I should add `(defaults to 'public')` in a comment?

Reply:
Yup, which you've done. Though, rereading this, we should probably start following Kubernetes convention and CamelCase these and go with `InternetFacing` and `Internal` as the two options.

Reply:
The upstream issue for 1.16 support: https://github.com/kubernetes-sigs/cluster-api/issues/1442